src/coins.h

```diff
@@ -162,26 +162,19 @@ struct CCoinsCacheEntry
     //! Adding a flag also requires a self reference to the pair that contains
     //! this entry in the CCoinsCache map and a reference to the sentinel of the
     //! flagged pair linked list.
-    inline void AddFlags(uint8_t flags, CoinsCachePair& self, CoinsCachePair& sentinel) noexcept
+    static inline void AddFlags(uint8_t flags, CoinsCachePair& self, CoinsCachePair& sentinel) noexcept
```

Comment:
I think `self` is no longer a good name for this parameter if we make this static. It should just be `pair`.

Same for `SetDirty` and `SetFresh`.

Reply:
Done